src/datasets/non_desktop/apple_app_store/reference.md

```diff
@@ -0,0 +1,115 @@
+# Apple App Store
+<!-- toc -->
+
+# Introduction
+The [`apple_app_store`](https://console.cloud.google.com/bigquery?project=moz-fx-data-marketing-prod&folder=&organizationId=&p=moz-fx-data-marketing-prod&d=apple_app_store&page=dataset) dataset is used to understand the acquisition performance for the nondesktop products through the Apple App Store along the key metrics and dimensions. Apple’s documentation for all metrics and dimensions can be found [here](https://help.apple.com/app-store-connect/#/itc21781223f).
```

Comment:
I try really hard not to just use "here" for hyperlinks. There's a bunch of essays on this online, including this one:

https://medium.com/@heyoka/dont-use-click-here-f32f445d1021

```suggestion
The [`apple_app_store`](https://console.cloud.google.com/bigquery?project=moz-fx-data-marketing-prod&folder=&organizationId=&p=moz-fx-data-marketing-prod&d=apple_app_store&page=dataset) dataset is used to understand the acquisition performance for mobile products on the Apple App Store along the key metrics and dimensions. Apple’s documentation for all metrics and dimensions can be found [in the app store connect reference](https://help.apple.com/app-store-connect/#/itc21781223f).
```